horizon/responses.go

```diff
@@ -31,6 +31,16 @@ type Account struct {
 	Signers              []Signer          `json:"signers"`
 }
 
+func (a Account) GetNativeBalance() string {
```

Comment:
OK, I changed it to return 0 because if there is no native balance then it means a user has 0 XLM. But still, horizon will never return balance array without native asset.